.eslintrc.json

```diff
@@ -111,6 +111,7 @@
       {
         "ts-nocheck": false
       }
-    ]
+    ],
+    "@typescript-eslint/explicit-module-boundary-types": "off"
```

Comment:
I still see some disabling of this in mockRequestUtils.ts, can they be removed now this is added?

Reply:
Done

Reply:
hmm didn't work; will return to this